app/models/details/transformation/TransformedElement.scala

```diff
@@ -16,17 +16,111 @@
 
 package models.details.transformation
 
-import viewmodels.AnswerRow
+import models.details._
+import utils.CountryOptions
+import viewmodels.{AnswerRow, SuperSection}
+
 import scala.language.implicitConversions
 
 trait TransformedElement[I] {
 
+  val entityType: String = "individual"
```

Comment:
Make private/protected anything not needing to be public

Reply:
fixed